### PR TITLE
updated the project technical outline template

### PR DESCRIPTION
**Pull request by oindrillac**

### CODE REVIEW

File: project-structure/project-document-template.md [hunk 1]
```diff
@@ -1,70 +1,47 @@
...
-impact this project will have if successfully completed._
-
-
-
-1. Situation and current issues
-2. Key Questions
-3. Hypothesis: Overview of how it could be done
-4. Impact
-
+_In this document, based on the available project outline and summary of the project pitch, to the best of your abilities, you will come up with the technical plan or goals for implementing the project such that it best meets the stakeholder requirements.
```
Comment (trgardos) [inline project-structure/project-document-template.md:7]: Leading underscore? Did you mean to have a trailing underscore to italicize?

Comment (oindrillac) [reply project-structure/project-document-template.md:7]: Yes, let me fix that

Comment (oindrillac) [reply project-structure/project-document-template.md]: done

File: project-structure/project-document-template.md [hunk 2]
```diff
@@ -1,70 +1,47 @@
...
+*To assist in outlining the steps needed to achieve our final goal, outline the AI-less process that we are trying to
+automate with Machine Learning. Provide as much detail as possible.*
 
+### B. Problem Statement:
 
-### B. Checklist for project completion
-
-_Provide a bulleted list of the concrete deliverables and artifacts that, when complete, define the completion of the
- project._
+*In as direct terms as possible, provide the “Data Science” problem statement version of the overview. Think of this as
```
Comment (trgardos) [inline project-structure/project-document-template.md:16]: maybe...

> "Data Science" or "Machine Learning" problem statement...

File: project-structure/project-document-template.md [hunk 3]
```diff
@@ -1,70 +1,47 @@
...
+translating the above into a more technical definition to execute on. eg: a classification problem to segregate users into one of three groups on based on the historical user data available from a publicly available database*
 
+### C. Checklist for project completion
 
+*Provide a bulleted list of the concrete techinal goals and artifacts that, when complete, define the completion of the
```
Comment (trgardos) [inline project-structure/project-document-template.md:21]: Maybe...

> Provide a bulleted list, to the best of your current understanding, of the concrete...

File: project-structure/project-document-template.md [hunk 4]
```diff
@@ -1,70 +1,47 @@
...
+project.*
```
Comment (trgardos) [inline project-structure/project-document-template.md:22]: > project. This checklist will likely evolve as your project progresses.

File: project-structure/project-document-template.md [hunk 5]
```diff
@@ -1,70 +1,47 @@
...
 
-
-
 1. 
 
-
 ## Weekly Meeting Updates
 
-_Keep track of ongoing meetings in a google doc and link it to this markdown_
+*Keep track of ongoing meetings in your project desciption document or maintain a separate google doc and link it to this markdown*
```
Comment (trgardos) [inline project-structure/project-document-template.md:45]: I thin Lydia was pretty adamant that the students keep the meeting notes in the Project Description document.

Comment (trgardos) [reply project-structure/project-document-template.md:45]: Maybe something like

> Keep track of ongoing meetings in the Project Description document prepared by Spark staff for your project.

File: project-structure/project-document-template.md [hunk 6]
```diff
@@ -1,70 +1,47 @@
...
 
-Example meeting notes [google doc](https://docs.google.com/document/d/1EQbzWQMvNOFHZhj7xVXqPxkXgTaWG6kHKtYo5y-USVo/edit?usp=sharing). You can create a copy of this document for your project and update the meeting document as the project progresses.
+Example meeting notes [google doc](https://docs.google.com/document/d/1WRimK8O1iTdC-5t2v_mXN5uc0E72TicQ6VJTzaieWhU/edit?usp=sharing). You can create a copy of this document for your project and update the meeting document as the project progresses.
```
Comment (trgardos) [inline project-structure/project-document-template.md:47]: Then we can delete this line.

File: project-structure/project-document-template.md (hunk 2 above)
Comment (oindrillac) [reply project-structure/project-document-template.md:16]: I like that

File: project-structure/project-document-template.md (hunk 3 above)
Comment (oindrillac) [reply project-structure/project-document-template.md:21]: yeah, fair

File: project-structure/project-document-template.md (hunk 4 above)
Comment (oindrillac) [reply project-structure/project-document-template.md:22]: yep, will add that. should be a changing list

File: project-structure/project-document-template.md (hunk 5 above)
Comment (oindrillac) [reply project-structure/project-document-template.md:45]: yes, consolidating them in one place should work best, will update that. Maybe at the end of this markdown will also add that the contents of this doc should be appended to the project descriptions doc.

**Comment by oindrillac** [comment]
Thanks for your review @trgardos. Updated as per your comments

**Review by trgardos — APPROVED** [review]

**Comment by trgardos** [comment]
Thanks for updating. Looked great!

**Pull request merged**